frontend/src/components/module-viewer/content/blocks/column/ColumnContent.tsx

```diff
@@ -0,0 +1,145 @@
+  editable?: boolean;
+}): React.ReactElement => {
+  const [isHovered, setIsHovered] = useState(false);
+  const {
+    isOpen: isEditModalOpen,
+    onOpen: onEditModalOpen,
+    onClose: onEditModalClose,
+  } = useDisclosure();
+  const {
+    isOpen: isDeleteModalOpen,
+    onOpen: onDeleteModalOpen,
+    onClose: onDeleteModalClose,
+  } = useDisclosure();
+  const context = useContext(EditorContext);
+
+  if (!context) return <></>;
+  const { dispatch } = context;
+
+  const deleteColumn = () => {
+    const content = {
+      [side]: null,
+    } as ColumnBlockState["content"];
+    dispatch({
+      type: "update-block",
+      value: { index, content },
+    });
+    onDeleteModalClose();
+  };
+
+  const editColumn = <T extends ContentBlockState>(
+    sideUpdates: T["content"],
+  ) => {
+    const content = {
+      [side]: {
+        ...block,
+        content: {
+          ...block?.content,
+          ...sideUpdates,
+        },
+      },
+    } as ColumnBlockState["content"];
+    dispatch({
+      type: "update-block",
+      value: { index, content },
+    });
+    onEditModalClose();
+  };
+
+  if (editable && block === null) {
+    return (
+      <Droppable droppableId={`${columnID} ${side} column"`}>
+        {(provided) => (
+          <Box
+            ref={provided.innerRef}
+            borderWidth="3px"
+            borderColor="grey.200"
+            margin="1rem"
+            width="100%"
+            minHeight="20rem"
+          />
+        )}
+      </Droppable>
+    );
+  }
+  if (editable && block !== null) {
+    return (
+      <Box
+        borderWidth="3px"
+        borderColor={isHovered ? "grey.200" : "white"}
+        margin="1rem"
+        width="100%"
+      >
+        {block && (
+          <VStack
+            width="100%"
+            height="100%"
+            padding="0.5rem 1rem"
+            spacing={2}
+            onMouseEnter={() => setIsHovered(true)}
+            onMouseLeave={() => setIsHovered(false)}
+          >
+            <Flex width="100%" alignItems="end" direction="column">
+              <EditContentOptionsMenu
+                isVisible={isHovered}
+                onEditClick={onEditModalOpen}
+                onDeleteClick={onDeleteModalOpen}
+              />
+              <Center w="100%" padding=".2rem">
+                {CONTENT_BLOCKS.render({ block, index, editable })}
+              </Center>
+            </Flex>
+            {EDIT_MODALS.render({
+              isOpen: isEditModalOpen,
+              onClose: onEditModalClose,
+              block,
+              onSave: editColumn,
+            })}
+            <DeleteModal
+              name="Content Block"
+              isOpen={isDeleteModalOpen}
+              onConfirm={deleteColumn}
+              onCancel={onDeleteModalClose}
+            />
+          </VStack>
+        )}
+      </Box>
+    );
+  }
+  // If not editable, return just content block
+  return (
+    <Center w="100%" padding=".2rem">
```

Comment:
Any reason not to use px instead of rem here? Just wondering since it's such a small padding.